docs/whats-new/preview-4.md

```diff
@@ -0,0 +1,585 @@
+---
+title: .NET Aspire preview 4
+description: .NET Aspire preview 4 is now available and includes many improvements and new capabilities
+ms.date: 03/04/2024
+---
+
+# .NET Aspire preview 4
+
+.NET Aspire preview 4 introduces significant enhancements across various parts of the stack, including addressing highly requested features from the community. Notable focus areas include improvements Entity Framework components, Podman support, and changes to the application model to easily choose between using existing resources or provisioning new ones.
+
+The .NET Aspire preview 4 version is `8.0.0 preview.4.24129.7`.
+
+## Podman support
+
+.NET Aspire preview 4 introduces support for running applications using Podman. Podman is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System. It's a great alternative to Docker for Linux users who want to run containers without a daemon.
+
+Docker or Podman will be auto-detected; if both are present, Docker is preferred. Podman can be explicitly enabled/forced via an environment variable `DOTNET_ASPIRE_CONTAINER_RUNTIME=podman`.
+
+## Dashboard
+
+The [.NET Aspire dashboard](../fundamentals/dashboard.md) has been really well received by the developer community. With preview 4, we've made several improvements to the dashboard to make it easier to use and more accessible.
+
+### Dashboard user experience updates
+
+:::image type="content" source="media/preview-4/dashboard-face-lift.png" lightbox="media/preview-4/dashboard-face-lift.png" alt-text=".NET Aspire Dashboard: Updates showing landing page.":::
+
+The dashboard has been updated with a new look and feel. The new dashboard is designed to reduce the space used by the navigation tabs and to make it easier to navigate between logs, metrics, and traces.
+
+### Run the .NET Aspire dashboard standalone
+
+The .NET Aspire dashboard can now be run as a standalone container image. This makes it easier to use the dashboard to manage applications that are running on a different machine or in a different environment. The dashboard can be used as an [OTLP](https://opentelemetry.io/docs/specs/otlp/) collector and viewer for applications that want to send and visualize telemetry data.
+
+Here's the command you can run to start the dashboard:
+
+```docker
+docker run --rm -it \
+  -p 18888:18888 \
+  -p 4317:18889 \
+  -d --name aspire-dashboard \
+  mcr.microsoft.com/dotnet/nightly/aspire-dashboard:8.0.0-preview.4
+```
+
+:::image type="content" source="media/preview-4/dashboard-as-container.png" lightbox="media/preview-4/dashboard-as-container.png" alt-text=".NET Aspire Dashboard: running in Docker Desktop.":::
+
+There are two ports that are exposed:
+
+1. `18888`: The port that serves the dashboard UI.
+1. `18889`: The port that serves the OTLP grpc endpoint.
```

Comment:
```suggestion
1. `4317`: The port that serves the OTLP grpc endpoint.
```

Reply:
https://docs.docker.com/network/

Reply:
These are the container ports.

Reply:
Ok. I think the doc here could be confusing because the container has those ports, but the exposed ports on the host from the earlier docker run command are different.

Reply:
I have no problem changing this doc. Do we have a container doc?

Reply:
I only know about the sample of using it.

Reply:
https://github.com/dotnet/docs-aspire/issues/471